pkg/kubelet/winstats/perfcounters.go

```diff
@@ -34,6 +34,8 @@ const (
 	// Perf counters are updated every second. This is the same as the default cadvisor collection period
 	// see https://github.com/google/cadvisor/blob/master/docs/runtime_options.md#housekeeping
 	perfCounterUpdatePeriod = 1 * time.Second
+	// defaultCachePeriod is the default cache period for each cpuUsage.
+	defaultCachePeriod = 15 * time.Second
```

Comment:
Would 15s too long compared to other metrics? Note that `perfCounterUpdatePeriod` is 1 seconds.

Reply:
cAdvisor uses 15s as the time interval, and we want to be consistent with cAdvisor in here.

Reply:
@dashpole do you now where this is set in cadvisor?

Reply:
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cadvisor/cadvisor_linux.go#L63

it is between 10 and 15 seconds

Reply:
Let's do 10s and add a comment here saying that this matches the cadvisor setting. 

10s also matches the period we use for containers.

Reply:
Done.